clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/JobController.groovy

```diff
@@ -45,6 +45,7 @@ class JobController {
   @Autowired
   FiatPermissionEvaluator fiatPermissionEvaluator
 
+  @PreAuthorize("hasPermission(#application, 'APPLICATION', 'READ') and hasPermission(#account, 'ACCOUNT', 'READ')")
```

Comment:
Is this something we want to do at the same time we remove all of this code?

Reply:
Can def pop in a separate PR if that makes more sense to y'all! (Currently it is just in its own commit.)

Reply:
IMO, that's a cleaner, more atomic, change.

Reply:
Removed and will be coming soon in a different PR!